### ISSUE TITLE
Error opening indexed db when syncing back changes

### ISSUE THREAD

**Issue by maurerle** [jhass/nextcloud-keeweb#157]
Opening a file editing, and syncing gives 
`Error opening indexed db`
on latest 0.6.3 release.

I think this is due to the removed caching/service worker?

**Comment by arnowelzel** [comment]
Please check if this is also the case for 0.6.2 - because as far I know the service worker fails there as well. That was the reason why I removed it completely as the intitialization did not succeed in Keeweb 0.6.2 as well also see https://github.com/jhass/nextcloud-keeweb/issues/154#issuecomment-678695611

**Comment by maurerle** [comment]
I 0.6.2 i can't access the app from incognito mode at all in Firefox because of 
`KdbxError: Error NotImplemented: AES-CBC not implemented`

Edge complains on save with "can't open ..." too, so this should not stop from releasing.

Even though I'd like to warn the user that he can't save before he edits a lot and realizes it then (or not at all)..

**Comment by jhass** [comment]
I don't think your suggestions are something we can or should solve here, rather than upstream.

**Comment by arnowelzel** [comment]
> 
> 
> I don't think your suggestions are something we can or should solve here, rather than upstream.

I agree. Syncing generally works - so it seems the service worker is not required to make syncing possible.

`2020-08-31T12:45:54.697Z [storage-cache] Error opening indexed db`

I believe this is caused by the missing browser storage in incognito mode. Also see https://github.com/keeweb/keeweb/issues/400

**Comment by maurerle** [comment]
I agree too.
Maybe a disclaimer in the README and https://apps.nextcloud.com/apps/keeweb that incognito and Old Edge is not supported would be the way of least frustration without bundling development ressources into those issues?
